build/Targets/GenerateCompilerExecutableBindingRedirects.targets

```diff
@@ -5,7 +5,7 @@
   <PropertyGroup>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
   </PropertyGroup>
-
+  
```

Comment:
@leppie not yet, but the current POR is that VS "15" will based on .editorconfig files.